src/plugins/condition/components/ConditionSet.vue

```diff
@@ -54,12 +54,19 @@ export default {
     },
     mounted() {
         this.conditionSetIdentifier = this.openmct.objects.makeKeyString(this.domainObject.identifier);
+        this.provideTelemetry();
+    },
+    beforeDestroy() {
+        if (this.stopProvidingTelemetry) {
+            this.stopProvidingTelemetry();
+        }
     },
     methods: {
         updateCurrentOutput(currentConditionResult) {
-            if (this.openmct.objects.makeKeyString(currentConditionResult.id) === this.conditionSetIdentifier) {
-                this.currentConditionOutput = currentConditionResult.output;
-            }
+            this.currentConditionOutput = currentConditionResult.output;
+        },
+        provideTelemetry() {
+            this.stopProvidingTelemetry = this.openmct.telemetry.subscribe(this.domainObject, output => output);
```

Comment:
Did you mean this to be `output => this.updateCurrentOutput(output)` ?

Reply:
@shefalijoshi YEEEEES. This is why I'm getting dupes. as you say I don't need to use ConditionManager to listen for updates in two places.

Fixed.